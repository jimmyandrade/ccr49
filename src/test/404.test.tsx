import {create} from "react-test-renderer";
import NotFoundPage from "../pages/404";

jest.mock("next/image", () => "Image");

describe("404 error page", () => {
    it("should match snapshot", () => {
        const testRenderer = create(<NotFoundPage />);
        expect(testRenderer).toMatchSnapshot();
    })
})