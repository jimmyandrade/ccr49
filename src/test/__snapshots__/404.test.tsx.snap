// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 error page should match snapshot 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthXs"
>
  <h1
    className="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom"
  >
    PÃ¡gina nÃ£oÂ encontrada
  </h1>
  <p
    className="MuiTypography-root MuiTypography-h4 MuiTypography-paragraph"
  >
    Desculpe
     
    <span
      aria-label="Emoji pensativo"
      role="img"
    >
      ğŸ˜”
    </span>
     
    a pÃ¡gina que vocÃª procura nÃ£o foi encontrada.
  </p>
  <a
    aria-disabled={false}
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined"
    href="/"
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
  >
    <span
      className="MuiButton-label"
    >
      Voltar ao inÃ­cio
    </span>
  </a>
</main>
`;
